stable/cluster-autoscaler/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 1.1.0
+version: 2.0.0
```

Heads up this will likely need to change again, as https://github.com/helm/charts/pull/15695 is on its way to getting merged.

```suggestion
version: 3.0.0
```

---

Thanks for the heads up!